.circleci/postgres-13.0-alpine-postgis/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# See https://github.com/appropriate/docker-postgis/issues/157#issuecomment-562812857
+
+FROM alpine:latest
+
+FROM circleci/postgres:13.0-postgis
+
+# this can be run like docker run -e POSTGRES_PASSWORD=randompassword (image-tag) for the user "postgres"
+
+# Keeping this here if we still need postgis v 3
```

Comment:
@tirumaraiselvan just the above statements were enough to setup postgres on alpine and also install postgis v3 as you can see here from this file. https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/postgres/images/13.0/postgis/Dockerfile

would this be enough or do we need to provide more details?